project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
spyscope 0.1.4 wants clj-time 0.5.0.  spyscope 0.1.5 wants clj-time 0.7.0.  I think we currently want clj-time 0.5.1 for our TK-related projects.

I can definitely add an explicit dep in the tk-jetty9 project deps for clj-time 0.5.1 and that works with a dependency on spyscope 0.1.5 so long as I add an exclusion for spyscope on clj-time:

```
(defproject puppetlabs/trapperkeeper-webserver-jetty9 "1.1.2-SNAPSHOT"
...
  :dependencies [[org.clojure/clojure "1.6.0"]
                 [org.clojure/tools.logging "0.2.6"]
                 [clj-time "0.5.1"]
...
    :profiles {:dev {
...
      :dependencies
        [spyscope "0.1.5" :exclusions [clj-time]]
```

Are you saying there's a way to have spyscope 0.1.5 explicitly include clj-time 0.7.0 and still have the rest of the project use clj-time 0.5.1?  I was thinking only one clj-time version could ultimately be in use.  Sorry, I'm probably misunderstanding what you're saying...